### ISSUE TITLE
Call to v21 AXAPI slb.virtual_server.vport.get() results in TypeError

### ISSUE THREAD

**Issue by bosatsu** [a10networks/acos-client#180]
**Issue:**
Calling slb.virtual_server.vport.get() using v21 client results in a type error.

**Traceback:**
```
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/jjmanzer/development/thor_api/.venv/src/acos-client/acos_client/v21/slb/virtual_port.py", line 105, in get
    if len(filtered_vports) > 0:
TypeError: object of type 'filter' has no len()
```

**Steps to reproduce:**
```
import acos_client as acos
api = acos.Client(ip_address, acos.AXAPI_21, username, password)
api.slb.hm.create('healthcheck1', api.slb.hm.HTTP, 5, 5, 5, 'GET', '/', '200', 80)

api.slb.service_group.create('pool1',
                             protocol=api.slb.service_group.TCP,
                             hm_name='healthcheck1',
                             lb_method=api.slb.service_group.LEAST_CONNECTION)

api.slb.server.create('server1', '2.1.1.7',
                           conn_limit=100,
                           conn_resume=90)

api.slb.server.port.create('server1', 80, api.slb.service_group.TCP,
                           conn_limit=100,
                           conn_resume=90)

api.slb.service_group.member.create('pool1', 'server1', 80)

api.slb.virtual_server.create('testvserver1', '1.1.1.5',
                              vrid=1,
                              template_virtual_server='ICMP-RATE-LIMIT')

api.slb.virtual_server.vport.create('testvserver1', '_2.1.1.3_TCP_80', api.slb.virtual_server.vport.HTTP, 80,
                                    service_group_name='pool1',
                                    conn_limit=100000,
                                    ha_conn_mirror=1,
                                    template_tcp='CONN-TIMEOUT-30S',
                                    no_dest_nat=0)

api.slb.virtual_server.vport.get('testvserver1', '_2.1.1.3_TCP_80', api.slb.virtual_server.vport.HTTP, 80)
```

**Comment by hthompson6** [comment]
Fixed in PR #181

**Comment by bosatsu** [comment]
Upon further investigation we found that this was not an actual and that the **Steps to reproduce** were flawed.  

When creating the `virtual_server` we did:
```
api.slb.virtual_server.vport.create('testvserver1', '_2.1.1.3_TCP_80', api.slb.virtual_server.vport.HTTP, 80,
                                    service_group_name='pool1',
                                    conn_limit=100000,
                                    ha_conn_mirror=1,
                                    template_tcp='CONN-TIMEOUT-30S',
                                    no_dest_nat=0)
```

specifically we used `api.slb.virtual_server.vport.HTTP` to create the virtual_server.  HTTP doesn't support no-dest-nat with out additional parameters, so in the case the acos_client was reporting correctly:

```
a10-ax3030-02-Active(config-slb vserver-vport)#service-group pool1
a10-ax3030-02-Active(config-slb vserver-vport)#no-dest-nat
a10-ax3030-02-Active(config-slb vserver-vport)#port 80 http
a10-ax3030-02-Active(config-slb vserver-vport)#no-des
a10-ax3030-02-Active(config-slb vserver-vport)#no-dest-nat
Port Type Mismatch
```
